.bundle/config

```diff
@@ -1,4 +1,2 @@
 ---
-BUNDLE_WITHOUT: omnibus_package
```

Comment:
Yeah ... just turns out the thing causes more issues than it solves. Taking that initial hit sucks some, but it gives you all the things and you can --without all day if you want.